st2actions/st2actions/policies/retry.py

```diff
@@ -63,15 +63,20 @@ def __init__(self, policy_ref, policy_type, retry_on, max_retry_count=2, delay=0
         self.max_retry_count = max_retry_count
         self.delay = delay or 0
 
-    def apply_before(self, target):
-        # Nothing to do here
-        target = super(ExecutionRetryPolicyApplicator, self).apply_before(target=target)
-        return target
-
     def apply_after(self, target):
-        target = super(ExecutionRetryPolicyApplicator, self).apply_before(target=target)
+        target = super(ExecutionRetryPolicyApplicator, self).apply_after(target=target)
```

Comment:
Good catch - I guess none of the existing tests caught it because it uses a default parent implementation which doesn't do anything right now (so mostly just out of luck and it would break if we changed the parent behavior).